core/README.md

```diff
@@ -0,0 +1,3 @@
+# Subxt-Core
+
+This library provides core functionality using in `subxt` and `subxt-signer`. It should be no-std compatible.
```

Comment:
I think for now we should add a warning here and in the lib docs that this is not stable may still change significantly between releases, so better to rely on `subxt` where possible: there's a lot to look at and think about with this new crate, but at the same time it'd be good to feel like we can release it now and iterate on it without worrying too much about breaking things :)